services/cards-publication/src/test/resources/application.yml

```diff
@@ -10,7 +10,7 @@ spring:
         #        class: io.confluent.kafka.serializers.KafkaAvroSerializer
         class: org.opfab.cards.publication.kafka.producer.KafkaAvroWithoutRegistrySerializer
   kafka:
-    bootstrap-servers: localhost:8384
+    bootstrap-servers: localhost:9092
```

Comment:
it should be possible to remove kafa config here i we remove unit test (see previous remarks)